contracts/src/v0.8/keystone/KeystoneForwarder.sol

```diff
@@ -213,20 +220,30 @@ contract KeystoneForwarder is IForwarder, ConfirmedOwner, TypeAndVersionInterfac
     }
   }
 
+  // solhint-disable-next-line chainlink-solidity/explicit-returns
   function _getMetadata(
     bytes memory rawReport
-  ) internal pure returns (bytes32 workflowId, uint32 donId, bytes32 workflowExecutionId, address workflowOwner) {
+  ) internal pure returns (bytes32 workflowExecutionId, uint32 donId, uint32 donConfigVersion, bytes2 reportId) {
+    // (first 32 bytes of memory contain length of the report)
+    // version                  // offset  32, size  1
+    // workflow_execution_id    // offset  33, size 32
+    // timestamp                // offset  65, size  4
+    // don_id                   // offset  69, size  4
+    // don_config_version,	    // offset  73, size  4
+    // workflow_cid             // offset  77, size 32
+    // workflow_name            // offset 109, size 10
+    // workflow_owner           // offset 119, size 20
+    // report_name              // offset 139, size  2
+    if (uint8(rawReport[0]) != 1) {
+      revert InvalidVersion(uint8(rawReport[0]));
```

Comment:
As discussed, we don't need this validation today. We can fix it in a follow-up.